.env

```
@@ -0,0 +1 @@
+BASE_URL=http://192.168.1.201:8000
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)